app/lib/qualifications_api/teacher.rb

```diff
@@ -21,18 +21,15 @@ def name
     def qualifications
       @qualifications = []
 
-      add_eyts
-      add_induction
-      add_itt
-      add_mandatory_qualifications
       add_npq
+      add_mandatory_qualifications
```

Comment:
Given we can have multiple mandatory qualifications returned, do we need explicit ordering by `awarded`, as we do for NPQ?

Reply:
Good spot, I hadn't noticed these could be multiples, have applied the same date sorting as others now.